react/ProductCustomizer.js

```diff
@@ -1,54 +1,180 @@
+import PropTypes from 'prop-types'
 import React, { Component } from 'react'
 import Modal from 'vtex.styleguide/Modal'
 import Button from 'vtex.styleguide/Button'
 
 import './global.css'
-import AttachmentsList from './components/AttachmentsList'
+import schema from './__mock__/index.json'
+import AddToCart from './components/Buttons/AddToCart'
+import VariationList from './components/Variation/List'
+import ChangeToppings from './components/Buttons/ChangeToppings'
+import ProductCustomizerService from './utils/ProductCustomizerService'
 
 const isMobile = window.__RUNTIME__.hints.mobile
```

Comment:
@lucasecdb There's no public export of that name on the 'render' global package, I think the only one available is an Render Prop Component already implementing `withRuntimeContext`, called `RenderContextConsumer`

Reply:
@hapoza It is being exported, as you can see [here](https://github.com/vtex-apps/render-runtime/blob/master/react/typings/global.d.ts#L277). There's an example on the [`FilterNavigator`](https://github.com/vtex-apps/search-result/blob/master/react/FilterNavigator.js#L5) component of the search-result, using this hoc.

Reply:
LOL I swear I logged all the exports on the browser and only saw `RenderContextConsumer`.
So, which one is the most recommended to use and why?

Reply:
I prefer to use `withRuntimeContext` because I can use the `runtime` prop outside of the render method, so if I need to call stuff like `runtime.navigate()`, but if you only need it in render I don't see much of an issue using the context consumer.